packages/backend/src/accounting/psql/service.test.ts

```diff
@@ -767,4 +775,234 @@ describe('Psql Accounting Service', (): void => {
       )
     })
   })
+
+  describe('createTransfer', (): void => {
+    let accountFactory: AccountFactory
+    let assetService: AssetService
+
+    beforeAll(async () => {
+      accountFactory = new AccountFactory(accountingService)
+      assetService = await deps.use('assetService')
+    })
+
+    describe.each`
+      sameAsset | description
+      ${true}   | ${'same asset'}
+      ${false}  | ${'cross-currency'}
+    `('$description', ({ sameAsset }): void => {
+      let sourceAccount: LiquidityAccount
+      let destinationAccount: FactoryAccount
+      const startingSourceBalance = 10n
+      const startingDestinationLiquidity = 100n
+      const timeout = 10_000n // 10 seconds
+
+      beforeEach(async (): Promise<void> => {
+        const sourceAsset = await assetService.create(randomAsset())
+        assert.ok(!isAssetError(sourceAsset))
+
+        sourceAccount = await accountFactory.build({
+          balance: startingSourceBalance,
+          asset: sourceAsset
+        })
```

Comment:
As a future update, I'd like to standardize account & transfer creation in these tests, for now, just wanted to make it work